examples/life-cycle-hooks.yaml

```diff
@@ -0,0 +1,96 @@
+    apiVersion: argoproj.io/v1alpha1
```

Comment:
this is a long example, and I think it makes it hard to understand how this works